App.vue

```diff
@@ -74,4 +74,9 @@ body {
     }
   }
 }
+
+.sf-input input, .sf-search-bar__input {
+  outline: none;
+  padding-left: 3px;
```

Comment:
What is the padding left fixing here @priyal-hotwax

Reply:
Gave spacing between border line of input and the text @dt2patel